--- a/requests-2.33.0.dist-info/RECORD
+++ b/requests-2.33.0.dist-info/RECORD
@@ -18,7 +18,7 @@
 requests/utils.py,sha256=xDMVThT3jebVDqf-dTgluLuzyUWUaN7gUvi2jtwlctw,33091
 requests-2.33.0.dist-info/licenses/LICENSE,sha256=CeipvOyAZxBGUsFoaFqwkx54aPnIKEtm9a5u2uXxEws,10142
 requests-2.33.0.dist-info/licenses/NOTICE,sha256=9REJct7a0rTp0xRRja87fXLW4C5Jms2AIYHeb3RXHcw,38
-requests-2.33.0.dist-info/METADATA,sha256=DMD6bom-g8-hd3lXUZ4__-eNGnmzBWGTTR5DhPsXdxE,5126
+requests-2.33.0.dist-info/METADATA,sha256=me2N_HIS9nbj7dlVTctEDENb4Iv-UUO39KdSoa0MlnU,5135
 requests-2.33.0.dist-info/WHEEL,sha256=aeYiig01lYGDzBgS8HxWXOg3uV61G9ijOsup-k9o1sk,91
 requests-2.33.0.dist-info/top_level.txt,sha256=fMSVmHfb5rbGOo6xv-O_tUX6j-WyixssE-SnwcDRxNQ,9
 requests-2.33.0.dist-info/RECORD,,
